x-pack/plugins/fleet/common/constants/agent.ts

```diff
@@ -5,8 +5,7 @@
  * 2.0.
  */
 
-export const AGENT_SAVED_OBJECT_TYPE = 'fleet-agents';
-export const AGENT_ACTION_SAVED_OBJECT_TYPE = 'fleet-agent-actions';
+export const AGENTS_PREFIX = 'fleet-agents';
```

Comment:
@nchaulet this constant is still used at many places as `kuery` parameter, so I only renamed it